src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+
+body {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+ width: 16px;
+ height: 16px;
+ background-repeat: no-repeat;
+ margin-right: 4px;
```

Comment:
last star shouldn't have `margin` to avoid margin collapsing. 
Use `:not(:last-child)` instead